examples/lib/stories/widgets/sprite_widget.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:dashbook/dashbook.dart';
 import 'package:flame/flame.dart';
 import 'package:flame/widgets.dart';
+import 'package:flutter/material.dart';
```

Comment:
These changes are from the other PR aren't them?

Reply:
Haha, you are too quick, I wrote in telegram at the same time as you wrote this